### PR TITLE
:sparkles: Update guilder-config.yaml - second host port to 8097

### PR DESCRIPTION
**Pull request by clubanderson**
and update all other references to port 8082 in doc

## Related issue(s)

Fixes #

### CODE REVIEW

**Comment by clubanderson** [comment]
@MikeSpreitzer does the special workload need to change from 8082 to 8097 also?

File: docs/content/Coding Milestones/PoC2023q1/example1.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ nodes:
 For the guilder cluster, which will get two workloads, create a file
 named `guilder-config.yaml` with the following contents.  The workload
 that uses hostPort 8081 goes in both clusters, while the workload that
-uses hostPort 8082 goes only in the guilder cluster.
+uses hostPort 8097 goes only in the guilder cluster.
```
Comment (MikeSpreitzer) [inline docs/content/Coding Milestones/PoC2023q1/example1.md:55]: This stays 8082.  Here `hostPort` is the kube jargon referring to the port on the node hosting the kube Pod, which is _inside_ the kind cluster not the host of the kind cluster.

File: docs/content/Coding Milestones/PoC2023q1/example1.md [hunk 2]
```diff
@@ -584,7 +584,7 @@ spec:
         ports:
         - name: http
           containerPort: 80
-          hostPort: 8082
+          hostPort: 8097
```
Comment (MikeSpreitzer) [inline docs/content/Coding Milestones/PoC2023q1/example1.md:587]: This stays 8082, as long as the `containerPort` in the kind Cluster stays 8082.  This is talking about the kube node port inside the kind cluster, not the host of the kind cluster.

File: examples/special-workload.yaml [hunk 3]
```diff
@@ -35,7 +35,7 @@ spec:
         ports:
         - name: http
           containerPort: 80
-          hostPort: 8082
+          hostPort: 8097
```
Comment (MikeSpreitzer) [inline examples/special-workload.yaml:38]: This again is a port number inside the kind cluster rather than on the host of the kind cluster, so stays 8082.

**Comment by MikeSpreitzer** [comment]
There is also this: https://kubernetes.slack.com/archives/C058SUSL5AA/p1684954850086279
The unversioned quickstart is referencing examples files in a versioned release.

**Comment by clubanderson** [comment]
I think I fixed your finds, please review again @MikeSpreitzer

File: examples/special-workload.yaml [hunk 4]
```diff
@@ -35,7 +35,7 @@ spec:
         ports:
         - name: http
           containerPort: 80
-          hostPort: 8082
+          hostPort: 8082
```
Comment (MikeSpreitzer) [inline examples/special-workload.yaml:38]: Somehow a trailing space crept in here.

**Pull request merged**